pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
```

Comment:
@smroid you may want to customize this section more, such as author email, keywords, classifiers, etc. up to you!